README.md

```diff
@@ -8,13 +8,31 @@ Small tool to convert an IAM Policy in JSON format into a Terraform [`aws_iam_po
 
 Download the latest binary from the [releases page](https://github.com/flosell/iam-policy-json-to-terraform/releases) and put it into your `PATH` under the name `iam-policy-json-to-terraform`
 
+    brew install iam-policy-json-to-terraform
```

Comment:
This is a bit out of context, looks like it belongs to the previous instruction ("download the latest binary...") instead of an alternative. Maybe we can format this a bit more explicitly that there are multiple options (`brew`, manual download, `go get`?

Reply:
I split it up with sections for OSX, other, and developer. Does that work?

Reply:
Looks good!